tests/Struct/FindologicProductTest.php

```diff
@@ -390,13 +389,20 @@ public function testProduct(): void
             $config
         );
 
+        $productKeyword = new Keyword('FINDOLOGIC001');
+        $keywords = $findologicProduct->getKeywords();
+        $this->assertNotCount(0, $keywords);
+        foreach ($keywords as $keyword) {
+            if ($keyword->getValue() === $productKeyword->getValue()) {
+                $this->assertEquals($productKeyword, $keyword);
```

Comment:
Having this inside the condition does not sure ensure it gets called. The test would still pass even if this condition is not met. Put a `boolean` flag as `false` by default outside the `foreach`  and if the condition is met, make it `true`.  Then after the `foreach` ensure that the flag is `assertTrue()`. That way we can be sure that the keyword was found.